integration_tests/snap.snapcraft.yaml

```diff
@@ -0,0 +1,5 @@
+---
+description: A test part with snap/snapcraft.yaml
+maintainer: John Doe <john.doe@example.com>
+parts: [snap-testpart]
+origin: lp:~joetalbott/+junk/snap-testpart
```

Comment:
Does this make anyone else a tad nervous? I know this is the pattern followed elsewhere in the parser, where everything seems to be in @josepht's junk... but at least in my own experience, that's only for throw-away stuff. If this is accidentally deleted our tests start failing.

Is there a better place to put this stuff?